pkg/storaged/fsys-panel.jsx

```diff
@@ -84,7 +84,7 @@ export class FilesystemsPanel extends React.Component {
                 name = vgroup.Name + "/" + lvol.Name;
 
             if (!name)
-                name = block_name(backing_block || block);
+                name = block.IdLabel || block_name(backing_block || block);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17668-20220824-105520-592ef826-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)